### DISCUSSION TITLE
UDM Unifi v4.0.6 support.

### DISCUSSION THREAD

**Discussion by pattika** [fabianishere/udm-iptv#330] [Q&A]
Will Unifi v4.0.6 be supported? Since i updated my UDM Base last week i get "MRT_INIT: Address already in use" meaning the kernel is not supported.

**Comment by fabianishere** [comment]
That error indicates that some form of IGMP Proxy is already running on your device. Have you checked if native IPTV support is enabled/disabled in the UniFi UI? You cannot run both at the same time.

**Reply by pattika** [reply]
Wow, I'm sorry, indeed it was on, on all networks. I am 100% sure it was off before the update last week.
Problem solved. Thank you very much.

**Comment by BM-GH1** [comment]
Since the upgrade to v4.0.6 IPTV with KPN doesn't work anymore. It has been running fine for a long time with the help of the script from @fabianishere, tnx for that! 

I've tried all kinds of things, but I'm lost.... This is the diagnose output:

=== Configuration ===
WAN Interface: eth8
WAN VLAN: 4 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 213.75.0.0/16  217.166.0.0/16
LAN Interfaces: br4
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
=== Service Logs ===
Jul 02 13:04:10 UDM-PRO udm-iptvd[1550211]: DEBUG[init_interface@208]: if_mtu = 1500
Jul 02 13:04:10 UDM-PRO udm-iptvd[1550211]: DEBUG[init_interface@214]: if_falgs = 4163
Jul 02 13:04:10 UDM-PRO udm-iptvd[1550211]: DEBUG[init_interface@178]: iptv's ipv4 address is 10.47.224.169
Jul 02 13:04:10 UDM-PRO udm-iptvd[1550211]: DEBUG[init_interface@195]: if_index = 2013
Jul 02 13:04:10 UDM-PRO udm-iptvd[1550211]: DEBUG[init_interface@208]: if_mtu = 1500
Jul 02 13:04:10 UDM-PRO udm-iptvd[1550211]: DEBUG[init_interface@214]: if_falgs = 4163
Jul 02 13:04:10 UDM-PRO udm-iptvd[1550211]: ERROR[main@619]: exiting.........
Jul 02 13:04:10 UDM-PRO IMPROXY[1550211]: ERROR[main@619]: exiting.........
Jul 02 13:04:10 UDM-PRO systemd[1]: udm-iptv.service: Main process exited, code=exited, status=1/FAILURE
Jul 02 13:04:10 UDM-PRO systemd[1]: udm-iptv.service: Failed with result 'exit-code'.

Does anyone know the solution?

**Comment by dmutsaers** [comment]
@BM-GH1 You reinstalled and reconfigured UDM-IPTV after upgrade to UniFi OS 4.06? And disabled native IGMP support as stated above?
FWIW, I'm running UDM-IPTV on my UXG-Lite with UniFi OS 4.06 without problems (KPN).

**Reply by BM-GH1** [reply]
Problem is solved, IPTV streaming was indeed enabled. Thanks!
